.ci/jobDSL/jobs/apm-ci/apm-shared/oblt-test-env/stack_docker_images.groovy

```diff
@@ -30,7 +30,7 @@ pipelineJob("apm-shared/oblt-test-env/stack-docker-images") {
     artifactDaysToKeep(-1)
   }
   triggers {
-    cron('H H(3-4) * * 1-5')
+    cron('0 */1 * * 1-5')
```

Comment:
```suggestion
    cron('@hourly')
```

Reply:
I understand that we want to do this even on weekends, right?